app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User
+  include DataMapper::Resource
+
+  property :id,              Serial
+  property :name,            String, :required => true
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)